babel/src/main/java/org/apache/calcite/sql/babel/SqlDateTimeAtTimeZone.java

```diff
@@ -30,32 +29,46 @@
 
 import java.util.List;
 
+/**
+ * A {@code SqlDateTimeAtTimeZone} is an AST node that describes
+ * the date time expression of At Time Zone.
+ */
 public class SqlDateTimeAtTimeZone extends SqlCall implements SqlExecutableStatement {
   public static final SqlSpecialOperator OPERATOR =
       new SqlSpecialOperator("AT TIME ZONE", SqlKind.OTHER);
 
   private final SqlNode dateTimePrimary;
-  private final SqlIdentifier timeZoneValue;
+  private final SqlNode displacementValue;
 
+  /**
+   * Creates a {@code SqlTranslateUsingCharacterSet}.
```

Comment:
Should this be SqlDateTimeAtTimeZone?

Reply:
Fix in https://github.com/googleinterns/calcite/pull/91/commits/043584683a05963e9c9bf5ec0b90f13175967577